bouncerscript/test/test_task.py

```diff
@@ -619,6 +619,15 @@ def test_check_path_matches_destination(product_name, path, raises):
         "firefox-nightly-latest-l10n",
         "firefox-nightly-latest-l10n-ssl"
     ], False
+), (
```

Comment:
Maybe worth adding another test with the following:
```
`firefox-nightly-msi-latest-ssl`
`firefox-nightly-msi-latest-l10n-ssl`
```
with `raises` set to `True`. 

AND

 ```
`firefox-nightly-msi-latest-ssl`
`firefox-nightly-msi-latest-l10n-ssl`
`firefox-nightly-latest`
```
with `raises` set to `True`

AND 

 ```
`firefox-nightly-msi-latest-ssl`
`firefox-nightly-latest`
```
with `raises` set to `True`